src/web/assets/cp/src/js/BaseElementIndex.js

```diff
@@ -1138,7 +1138,7 @@ Craft.BaseElementIndex = Garnish.Base.extend(
       this.setInstanceState('selectedSource', this.sourceKey);
       this.sourceSelect.selectItem($source);
 
-      Craft.cp.updateSidebarMenuLabel();
```

Comment:
Probably worth leaving this around as an alias for `updateContentHeading()`, just in case a plugin is calling it.

Reply:
I've re-added the `updateSidebarMenuLabel` function inside that object and called the new function inside of it.